src/fbp/image_geom.jl

```diff
@@ -288,7 +288,7 @@ function _down_round(
 ) where {D}
 	out = val ./ down
 	# for non-divisors make dim a multiple of 2
-	fun = out -> out == round(out) ? out : 2 * round(out / 2)
+	fun = out -> out == round(out) ? out : 2 * (out ÷ 2)
```

Comment:
FYI, in case you're not aware of the differences: `floor(Int, dim1/down)` returns an `Int` no matter what `dim1` or `down` is. On the other hand, the return type of `dim1 ÷ down` is not necessarily an `Int`. (For example, `2.5 ÷ 2 === 1.0`).

This could sometimes cause a performance issue, especially for functions with the existence of an if-branch. As summarised in the [performance tips](https://docs.julialang.org/en/v1/manual/performance-tips/#Write-%22type-stable%22-functions), `fun` should be type stable to get rid of the runtime performance overhead.

```julia
out = rand(Float64, 64, 64, 30);
f_unstable = out -> out == round(out) ? out : 2 * floor(Int, out/2)
f_stable = out -> out == round(out) ? out : 2 * (out ÷ 2)

@btime f_unstable.($out); # 431.927 μs (14 allocations: 960.58 KiB)
@btime f_stable.($out); # 409.557 μs (5 allocations: 960.16 KiB)
```

You can call `@code_warntype f_unstable(out)` and will see that the return type is inferred as `Union{Int, Float64}`, which is not type stable.

Reply:
@johnnychen94 thanks very much for these tips! i just learned about `÷` and got overly excited about using it, not realized about Float÷Int.  i'm about to submit a PR that i hope will address these issues.